src/libs/actions/Policy.ts

```diff
@@ -417,40 +437,51 @@ function removeMembers(accountIDs, policyID) {
                 oldPolicyName,
             },
         })),
-        ..._.map(optimisticClosedReportActions, (reportAction, index) => ({
+        ...optimisticClosedReportActions.map<OnyxUpdate>((reportAction, index) => ({
             onyxMethod: Onyx.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${workspaceChats[index].reportID}`,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${workspaceChats?.[index]?.reportID}`,
             value: {[reportAction.reportActionID]: null},
         })),
         ...announceRoomMembers.onyxFailureData,
     ];
     API.write(
```

Comment:
For each API.write and API.read please define parameters type like in other files:
```ts
    type ReconnectToReportParameters = {
        reportID: string;
    }; // this is defined inside of a function

    const parameters: ReconnectToReportParameters = {
        reportID,
    };

    API.write('ReconnectToReport', parameters, {optimisticData, successData, failureData});
```